source/collections.txt

```diff
@@ -75,14 +75,6 @@ a unique icon:
 
```

Comment:
I really like the "Databases" page at the top of this section. I do think it's quite natural to start talking about Collections and Views after you talk about databases (assuming someone is following the pages in linear top-down order), but I think the way the "Collections and Views" subsection is currently structured is a bit strange. Specifically:

- There's a "Collections and Views" _section_, but when you expand it it's only one page, which feels unnecessary. 

- The "Collections and Views" _page_ is titled "Collections and Views" but the vast majority of the content on the page is about collections, with one small section on "Views", and I think that section is redundant with the section on the actual "Views" page.

Some _possible_ changes to consider:

- Remove the "Views" section in the "Collections and Views" page and migrate it to the "Views" page, if necessary. 

- Remove the "Collections and Views" sections and just have single, separate "Collections" and "Views" pages, much like the "Databases" page. These two pages would go right underneath the "Databases" page in the following order: "Databases", "Collections", "Views".

- If we're really tied to the idea of the ">" expandable subsection, and want to double down on task-oriented documentation, it's possible we could create a  "> Collections" expandable subsection, and within that subsection, have individual pages that correspond to the sections in the "Collections and Views" page. So for example, you'd have ">Collections" as the expandable menu and then you'd have "Create a Collection", "Drop a Collection", etc. (like the "> Manage Documents" section. The pages in ">Collections" would be quite short though, but perhaps that's not that big of a deal.